### ISSUE TITLE
Can't include Pillar SLS across GitPillar repositories

### ISSUE THREAD

**Issue by eliasp** [saltstack/salt#27932]
When trying to include an SLS in a GitPillar SLS across git repositories, it won't be found:
```
minion-012345-test:
    _errors:
        - Specified SLS 'some.pillar' in environment 'base' is not available on the salt master
```
```
    Salt Version:
               Salt: 2015.8.0
     
    Dependency Versions:
             Jinja2: 2.7.2
           M2Crypto: Not Installed
               Mako: 0.9.1
             PyYAML: 3.10
              PyZMQ: 14.4.0
             Python: 2.7.6 (default, Jun 22 2015, 17:58:13)
               RAET: Not Installed
            Tornado: 4.2.1
                ZMQ: 4.0.4
               cffi: Not Installed
           cherrypy: Not Installed
           dateutil: 1.5
              gitdb: 0.5.4
          gitpython: Not Installed
              ioflo: Not Installed
            libnacl: Not Installed
       msgpack-pure: Not Installed
     msgpack-python: 0.3.0
       mysql-python: Not Installed
          pycparser: Not Installed
           pycrypto: 2.6.1
             pygit2: Not Installed
       python-gnupg: Not Installed
              smmap: 0.8.2
            timelib: Not Installed
     
    System Versions:
               dist: Ubuntu 14.04 trusty
            machine: x86_64
            release: 3.13.0-55-generic
             system: Ubuntu 14.04 trusty
```

**Comment by eliasp** [comment]
Fix available in #27933

**Comment by jfindlay** [comment]
@eliasp, thanks for fixing this.

**Comment by eliasp** [comment]
Fixed in f884df5

**Comment by jfindlay** [comment]
Nice, thanks, @eliasp.
